### PR TITLE
update trident template

### PR DESCRIPTION
**Pull request by nkkashyap**
Signed-off-by: Neeraj K Kashyap <nkashyap@in.ibm.com>

### CODE REVIEW

File: config-templates/netapp/netapp-trident/20.07/custom-parameters.json [hunk 1]
```diff
@@ -70,6 +70,6 @@
     "displayname": "User Password",
     "name": "user-pass",
     "required": "true",
-    "type": "text"
+    "type": "secret"
```
Comment (prgavali) [inline config-templates/netapp/netapp-trident/20.07/custom-parameters.json:73]: I think , we should add both username and password as `secret`

**Review by prgavali — CHANGES_REQUESTED** [review]
reviewed

File: config-templates/netapp/netapp-trident/20.07/custom-parameters.json (hunk 1 above)
Comment (nkkashyap) [reply config-templates/netapp/netapp-trident/20.07/custom-parameters.json:73]: how to differentiate between the filed one need obfuscation and the other one does not require obfuscation?

Comment (nkkashyap) [reply config-templates/netapp/netapp-trident/20.07/custom-parameters.json:73]: change secret type to `secret`

**Pull request merged**